tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
btw the reason I'm pushing for out of the box support for 16.04 is that there's a downstream [issue](https://github.com/geerlingguy/drupal-vm/issues/1251) in Drupal VM to add this role.